### PR TITLE
feat(components): [el-scrollbar] expose scrollTo method

### PR DESCRIPTION
**Pull request by msidolphin**
Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/scrollbar/src/scrollbar.vue [hunk 1]
```diff
@@ -97,6 +97,20 @@ export default defineComponent({
       }
     }
 
+    const isScrollToOptions = (
+      options: unknown
+    ): options is ScrollToOptions => {
+      return typeof options === 'object'
+    }
```
Comment (JeremyWuuuuu) [inline packages/components/scrollbar/src/scrollbar.vue:104]: Is it really necessary for extracting? I only saw 1 place that uses this

Comment (JeremyWuuuuu) [reply packages/components/scrollbar/src/scrollbar.vue:104]: Use `isObject` to do the judgement.

File: packages/components/scrollbar/src/scrollbar.vue [hunk 2]
```diff
@@ -97,6 +97,20 @@ export default defineComponent({
...
+
```
Comment (JeremyWuuuuu) [inline packages/components/scrollbar/src/scrollbar.vue:105]:
```suggestion
function scrollTo(xCord: number, yCord?: number): void
function scrollTo(options: ScrollToOptions): void
function scrollTo(arg1: unknown, arg2?: number) {
  if (isObject(arg1)) {
    wrap.value!.scrollTo(arg1)
   } else if (isNumber(arg1) && isNumber(arg2)) {
        wrap$.value!.scrollTo(arg1, arg2)
      }
}
```

**Review by jw-foss — COMMENTED** [review]
Left some comments for you 🏓

**Comment by msidolphin** [comment]
@JeremyWuuuuu done

**Review by jw-foss — APPROVED** [review]

**Pull request merged**